components/Cards/src/MDCCardCollectionCell.h

```diff
@@ -0,0 +1,158 @@
+
+/**
+ Returns the shadow elevation for an MDCCardViewState state
+
+ If no elevation has been set for a state, the value for MDCCardCellStateNormal will be returned.
+ Default value for MDCCardCellStateNormal is 1
+ Default value for MDCCardCellStateHighlighted is 8
+ Default value for MDCCardCellStateSelected is 8
+
+ @param state MDCCardCellStateNormal the card state
+ @return The shadow elevation for the requested state.
+ */
+- (MDCShadowElevation)shadowElevationForState:(MDCCardCellState)state;
+
+/**
+ Sets the border width for an MDCCardViewState state
+
+ @param borderWidth The border width
+ @param state MDCCardCellState the card state
+ */
+- (void)setBorderWidth:(CGFloat)borderWidth forState:(MDCCardCellState)state
+    UI_APPEARANCE_SELECTOR;
+
+/**
+ Returns the border width for an MDCCardCellState state
+
+ If no border width has been set for a state, the value for MDCCardCellStateNormal will be returned.
+ Default value for MDCCardCellStateNormal is 0
+ Default value for MDCCardCellStateHighlighted is 0
+ Default value for MDCCardCellStateSelected is 0
+
+ @param state MDCCardCellState the card state
+ @return The border width for the requested state.
+ */
+- (CGFloat)borderWidthForState:(MDCCardCellState)state;
+
+/**
+ Sets the border color for an MDCCardCellStateNormal state
+
+ @param borderColor The border color
+ @param state MDCCardCellState the card state
+ */
+- (void)setBorderColor:(nullable UIColor *)borderColor forState:(MDCCardCellState)state
+    UI_APPEARANCE_SELECTOR;
+
+/**
+ Returns the border color for an MDCCardCellStateNormal state
+
+ If no border color has been set for a state, the value for MDCCardCellStateNormal will be returned.
+ Default value for MDCCardCellStateNormal is clearColor
+ Default value for MDCCardCellStateHighlighted is clearColor
+ Default value for MDCCardCellStateSelected is clearColor
+
+ @param state MDCCardCellState the card state
+ @return The border color for the requested state.
+ */
+- (nullable UIColor *)borderColorForState:(MDCCardCellState)state;
+
+/**
+ Sets the shadow color for an MDCCardCellStateNormal state
+
+ @param shadowColor The shadow color
+ @param state MDCCardCellState the card state
+ */
+- (void)setShadowColor:(nullable UIColor *)shadowColor forState:(MDCCardCellState)state
+    UI_APPEARANCE_SELECTOR;
+
+/**
+ Returns the shadow color for an MDCCardCellStateNormal state
+
+ If no color has been set for a state, the value for MDCCardViewStateNormal will be returned.
+ Default value for MDCCardCellStateNormal is blackColor
+ Default value for MDCCardCellStateHighlighted is blackColor
+ Default value for MDCCardCellStateSelected is blackColor
+
+ @param state MDCCardCellState the card state
+ @return The shadow color for the requested state.
+ */
+- (nullable UIColor *)shadowColorForState:(MDCCardCellState)state;
+
+/**
+ The image for the selected state.
+ Default is the checked circle image.
+ */
+@property(nonatomic, strong, nullable) UIImage *selectedImage;
```

Comment:
Consider making this a UI_APPEARANCE_SELECTOR to allow clients to set a "default" image.

Reply:
+1 to UI_APPEARANCE_SELECTOR.